scripts/vcpkg_ports/axom/vcpkg.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Newer `vcpkg` is using `vcpkg.json` manifest files instead of `CONTROL` files within its ports